test/integration/commands/excel2json/test_resources.py

```diff
@@ -183,63 +189,49 @@ class TestValidateWithSchema:
     lambda _: e2j._validate_resources([])
 
     def test_invalid_super(self) -> None:
-        expected_msg = (
```

Comment:
In fact you have now removed the `test_invalid_super()` test, and replaced it by a new test. I think we need both of them: The `test_invalid_super()` should remain untouched. Your new test should be added as a test on its own.

Reply:
Now I see that I have put several mistakes into the file `testdata/invalid-testdata/excel2json/resources-invalid-super.xlsx`. We should create 2 different Excel files, one for every mistake.